+adi/+FMComms5/Base.m

```diff
@@ -0,0 +1,77 @@
+classdef (Abstract, Hidden = true) Base < adi.AD9361.Base    
+    
+    %adi.AD9361.Base Class
+    %   This class contains shared parameters and methods between TX and RX
+    %   classes
+    properties (Nontunable, Logical)
+        %EnableCustomFilter Enable Custom Filter
+        %   Enable use of custom filter file to set SamplingRate, 
+        %   RFBandwidth, and FIR in datapaths
+        EnableCustomFilterChipB = false;
+    end
+    
+    properties (Nontunable)
+        %CustomFilterFileName Custom Filter File Name
+        %   Path to custom filter file created from filter wizard
+        CustomFilterFileNameChipB = '';
+    end
+    
+    properties (Abstract)
+        %CenterFrequency Center Frequency
```

Comment:
Update all 3 parameters to include "Chip B" in docstrings. So CenterFrequency Center Frequency -> CenterFrequencyChipB Center Frequency Chip B